rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/db/RMNodeData.java

```diff
@@ -42,8 +42,12 @@
 import org.hibernate.annotations.Type;
 import org.hibernate.type.SerializableToBlobType;
 import org.objectweb.proactive.core.node.Node;
+import org.ow2.proactive.jmx.naming.JMXTransportProtocol;
 import org.ow2.proactive.resourcemanager.authentication.Client;
 import org.ow2.proactive.resourcemanager.common.NodeState;
+import org.ow2.proactive.resourcemanager.common.event.RMNodeDescriptor;
+import org.ow2.proactive.resourcemanager.common.event.RMNodeEvent;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.ow2.proactive.resourcemanager.common.event.RMNodeEvent'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)